dockstore-webservice/src/main/java/io/dockstore/webservice/core/WorkflowVersion.java

```diff
@@ -83,6 +83,14 @@
     @ApiModelProperty(value = "The subclass of this for services.", position = 103)
     private Service.SubClass subClass = null;
 
+    @Column(columnDefinition = "TEXT")
+    @ApiModelProperty(value = "JSON needed to create DAG in UI", position = 104)
```

Comment:
I'm thinking this should not have @ApiModelProperty and should instead have `@JsonIgnore`. This is an internal caching mechanism that probably shouldn't be exposed to the client, especially since its value won't always be set.